docs/designers-developers/developers/richtext.md

```diff
@@ -0,0 +1,119 @@
+# RichText Reference
+
+RichText is a component that allows developers to render a [`contenteditable` input](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Editable_content), providing users with the option to format block content to make it bold, italics, linked, or use other formatting.
+
+The RichText component is extremely powerful because it provides built-in functionality you won't find in other components:
+
+* **Consistent Styling in the Admin and Frontend:** The editable container can be set to any block-level element, such as a `div`, `h2` or `p` tag. This allows the styles you apply in style.css to more easily apply on the frontend and admin, without having to rewrite them in editor.css.
+* **Cohesive Placeholder Text:** Before the user writes their content, it's easy to include placeholder text that's already styled to match the rest of the block editor.
+* **Control Over Formatting Options:** It's possible to dictate exactly which formatting options you want to allow for the RichText field. For example, you can dictate whether to allow the user to make text bold, italics or both.
+
+Unlike other components that exist in the [Component Reference](/packages/components/README.md) section, RichText lives separately because it only makes sense within the block editor, and not within other areas of WordPress.
+
+## Property Reference
+
+For a list of the possible properties to pass your RichText component, [check out the component documentation on Github](/packages/block-editor/src/components/rich-text/README.md).
+
+## Core Blocks Using the RichText Component
+
+There are a number of core blocks using the RichText component. The JavaScript edit function linked below for each block can be used as a best practice reference while creating your own blocks.
+
+* **[Button](/packages/block-library/src/button/edit.js):** RichText is used to enter the button's text.
+* **[Heading](/packages/block-library/src/heading/edit.js):** RichText is used to enter the heading's text.
+* **[Quote](/packages/block-library/src/quote/edit.js):** RichText is in two places, for both the quotation and citation text.
+* **[Search](/packages/block-library/src/search/edit.js):** RichText is used in two places, for both the label above the search field and the submit button text.
```

Comment:
Let's make these four URLs absolute URLs as well.

Reply:
Nope, let’s keep them 😅

Reply:
Or maybe not 🙁 I don’t know anymore. See my other comments.

Reply:
Let me know which approach you all want me to use on this front. I had followed the guidelines on https://github.com/WordPress/gutenberg/blob/master/docs/contributors/document.md#using-links, but I'm happy to make these absolute URLs if that makes more sense.

Reply:
Same as https://github.com/WordPress/gutenberg/pull/15956/files#r290299752 - These should be absolute URLs, as we don't expose the source code in the handbook.

Reply:
Great, it sounds like we should go ahead and make these absolute links. I can take care of that.

Reply:
These links have been updated to be absolute URLs. I'll go ahead and mark this as resolved.